### ISSUE TITLE
issues on running metrics-uptime

### ISSUE THREAD

**Issue by albdv** [sensu-plugins/sensu-plugins-uptime-checks#3]
Got the following exception while trying to run:

root@monitoring:# /usr/local/bin/metrics-uptime.py 
/usr/local/bin/metrics-uptime.py:23:in `load': /var/lib/gems/2.1.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:24: syntax error, unexpected ':' (SyntaxError)
/var/lib/gems/2.1.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:39: syntax error, unexpected ':'
/var/lib/gems/2.1.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:54: syntax error, unexpected ':'
/var/lib/gems/2.1.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:56: syntax error, unexpected ':', expecting keyword_end
/var/lib/gems/2.1.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:57: syntax error, unexpected ':', expecting keyword_do_cond or ';' or '\n'
/var/lib/gems/2.1.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:58: syntax error, unexpected ')', expecting '='
/var/lib/gems/2.1.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:63: syntax error, unexpected ':'
/var/lib/gems/2.1.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:78: syntax error, unexpected ':', expecting keyword_then or ';' or '\n'
/var/lib/gems/2.1.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:79: syntax error, unexpected end-of-input, expecting keyword_end
    from /usr/local/bin/metrics-uptime.py:23:in`<main>'

**Comment by teadur** [comment]
I get similar results with embedded ruby on debian jessie

root@n1:~# /opt/sensu/embedded/bin/metrics-uptime.py
/opt/sensu/embedded/bin/metrics-uptime.py:23:in `load': /opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:24: syntax error, unexpected ':' (SyntaxError)
/opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:39: syntax error, unexpected ':'
/opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:54: syntax error, unexpected ':'
/opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:56: syntax error, unexpected ':', expecting keyword_end
/opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:57: syntax error, unexpected ':', expecting keyword_do_cond or ';' or '\n'
/opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:58: syntax error, unexpected ')', expecting '='
/opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:63: syntax error, unexpected ':'
/opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:78: syntax error, unexpected ':', expecting keyword_then or ';' or '\n'
/opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py:79: syntax error, unexpected end-of-input, expecting keyword_end
        from /opt/sensu/embedded/bin/metrics-uptime.py:23:in`<main>'

**Comment by analytically** [comment]
What python version are you using? I just ran it on Ubuntu 15.04, python 2.7.9.

```
.../sensu-plugins-uptime-checks/bin$ ./metrics-uptime.py 
uptime.idletime 999874 1443254512
uptime.uptime 235031 1443254512
```

**Comment by teadur** [comment]
I'm running python 2.7.9 on debian jessie where it brakes.
Altought i have other boxes with same python version where it works correctly, it must be somekind of machine specific problem, ill try to test some more to figure out the exact culprit what brakes it.

**Comment by frederic-joventures** [comment]
python version 2.7.6
Same error for me on Ubuntu 14.04 TLS when called from 
/opt/sensu/embedded/bin/metrics-uptime.py

BUT 
run fine from:
/opt/sensu/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugins-uptime-checks-0.0.3/bin/metrics-uptime.py
uptime.idletime 541341 1447859471
uptime.uptime 67946 1447859471

**Comment by teadur** [comment]
it seems that  /opt/sensu/embedded/bin/metrics-uptime.py try's to load the python file as ruby file,
if i replace load with exec it works.

**Comment by eheydrick** [comment]
Looks like a binstubs issue. Maybe we need to include .rb binstubs that exec's the python?

**Comment by eheydrick** [comment]
We need to add ruby wrappers for the non-ruby plugins.

**Comment by analytically** [comment]
I've added a wrapper, but there's a PR to replace the python script.

**Comment by eheydrick** [comment]
The ruby version was released in 0.0.4. Can you try that.
